coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -467,6 +467,11 @@ func (s *server) acquireProtoJob(ctx context.Context, job database.ProvisionerJo
 		if err != nil {
 			return nil, failJob(fmt.Sprintf("get owner: %s", err))
 		}
+		var ownerSSHPublicKey, ownerSSHPrivateKey string
+		if ownerSSHKey, err := s.Database.GetGitSSHKey(ctx, owner.ID); err == nil {
```

Comment:
Should we explicitly handle errors here? (E.g. err no rows is fine). Otherwise an ephemeral error could trigger an inconsistency in builds.

Reply:
Good catch, added!